metrics/value.go

```diff
@@ -39,6 +54,22 @@ func (v *counterVal) Type() string {
 	return "counter"
 }
 
+func (v *counterVal) Keys(yield func(string) bool) {
+	for _, key := range []string{"count"} {
+		if !yield(key) {
+			return
+		}
+	}
+}
```

Comment:
The various `keys` implementation repeats a lot of boilerplate when the only difference is the input keys.

If we go with returning string slices for keys, let's just return slices directly.

If we go with returning iterators, would recommend having a helper function and then implementing various `Keys()` implementations as follows:

```suggestion
func stringSeq(strings ...string) iter.Seq[string] {
	return func(yield func(string) bool) {
		for _, key := range strings {
			if !yield(key) {
				return
			}
		}
	}
}

func (v *counterVal) Keys() iter.Seq[string] {
    return stringSeq("count")
}
```

Reply:
I attempted the `stringSeq` helper, but it allocates the slice of arguments on each call so I'm not sure it's worth it.